server/models/User.js

```diff
@@ -0,0 +1,79 @@
+import connection from '../helpers/connection';
+
+const client = connection();
+const users = [];
+export default class User {
+  constructor({
+    fullName,
+    email,
+    username,
+    password,
+  }) {
+    this.fullName = fullName;
+    this.email = email;
+    this.username = username;
+    this.password = password;
+  }
+
+  async save() {
+    const authQuery = 'INSERT INTO authentication(username, password) VALUES($1, $2) RETURNING *';
```

Comment:
Okay rather than just sending the password directly into the database, you should hash it
Good job though

Reply:
Yes I will. It's added in the `Description of Task to be completed?`. Thanks.